tests/compile/piecewise/test_simple.py

```diff
@@ -94,13 +94,14 @@ def test_simple_piecewise_compile():
         with set_compile_context([1, 2]):
             model(input_buffer)
```

Comment:
To double check: Here, `input_buffer` is not directly cached and reused by the backend, right?

Reply:
right. let me rename the variable. it is not a persistent buffer anymore. it can be a newly created tensor.